File: fibonacci/javascript_fibonacci/fibonacci.js
fibonacci = function( n ) {
  if( n == 0 ) {
    return 0;
  } else if( n == 1 ) {
    return 1;
  } else {
    return fibonacci(n - 1) + fibonacci(n - 2);
  }
}

//document.write('fibonacci(5)=');
//document.write(fibonacci(5));
for (idx_i = 1; idx_i < 25; idx_i++) {
  console.log("fibonacci(" + idx_i + ")=<" + fibonacci(idx_i) + ">");
}

File: fibonacci/rlang_fibonacci/fibonacci.R
#comment

fibonacci <- function(n) {
  if(n <= 1) {
    return(n)
  } else {
    return(fibonacci(n-1) + fibonacci(n-2))
  }
}

#sapply(seq(1,24),fibonacci)

main=function() {
  args = commandArgs(trailingOnly=TRUE)
  if (length(args) != 0) {
    #print("Usage: <noargs>")
    stop("Usage: example.R <noargs>", call.=FALSE)
  }

  for (idx_i in 1:24){
    cat(paste("fibonacci(", idx_i, ")=<", fibonacci(idx_i), ">\n", sep = ""))
  }
}

main()
